packaging/nix/umu-run.nix

```diff
@@ -1,18 +0,0 @@
-{
-  package,
-  buildFHSEnv,
-  writeShellScript,
-  version,
-  ...
-}:
-buildFHSEnv {
-  name = "umu-run";
-  version = "${version}";
-  targetPkgs = pkgs: [
-    package
-  ];
-  multiArch = true;
-  runScript = writeShellScript "umu-run-shell" ''
-    ${package}/bin/umu "$@"
-  '';
-}
```

Comment:
@LovingMelody  https://github.com/Open-Wine-Components/umu-launcher/pull/345#discussion_r1922854537

> PR submitted for upstream change: [NixOS/nixpkgs#375423](https://github.com/NixOS/nixpkgs/pull/375423)

As you can see, the existing implementation also doesn't support that, so I don't think it's in scope for this PR to add support.

Reply:
> I don't think it's in scope for this PR to add support.

Happy to be persuaded otherwise. Especially if https://github.com/NixOS/nixpkgs/pull/375423 is merged before this one.